helms/k8s-gke-service-account-assigner/templates/_helpers.tpl

```diff
@@ -0,0 +1,39 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "k8s-gke-service-account-assigner.name" -}}
```

Comment:
can we transform "k8s-gke-service-account-assigner" to shorter more convenient name?

Reply:
done in c03be05